cmd2/__init__.py

```diff
@@ -1,6 +1,14 @@
 #
 # -*- coding: utf-8 -*-
 """This simply imports certain things for backwards compatibility."""
-from .cmd2 import __version__, Cmd, Statement, EmptyStatement, categorize
+
+from pkg_resources import get_distribution, DistributionNotFound
+try:
+    __version__ = get_distribution(__name__).version
```

Comment:
So this is finally going to make me abandon my PYTHONPATH hack way of doing ``cmd2`` development and switch to the **pip** way instead using:
```sh
$ cd ~/src/cmd2
$ pip install -e .
```

But that is probably a good thing ...